src/App.js

```diff
@@ -2,19 +2,55 @@ import React, { Component } from 'react';
 import './App.css';
 
 class App extends Component {
-  state = { blocks: ['hello world', 'another string'] };
+  state = {
+    blocks: [
+      { type: 'heading', text: 'hello world' },
+      { type: 'text', text: 'another string' },
+      { type: 'image', url: 'http://fpoimg.com/300x300', text: 'FPO image' },
+      { type: 'blockQuote', text: 'this is a quote' },
+    ].map(this.withKeys),
+  };
+
+  withKeys(block, index) {
```

Comment:
Maybe the name of this function should be `toObjectsWithKey`? Just for consistency with `toComponents`? Up to you.